code/src/index.css

```diff
@@ -1,13 +1,144 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  width: auto;
+background-color: black;
+
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+main{
+  margin: 0 auto;
+  /* border: solid 2px blue; */
+}
+
+h1 {
+  color: #fff;
+  padding: 10px;
+  font-size: 24px;
+  margin: 0 12px;
+}
+
+a:link, a:visited {
+  color: white;
+}
+
+
+.albumsWrapper {
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: 10px;
+}
+
+.artistWrapper {
+  margin-top: 20px;
+}
+
+.albumWrapper {
```

Comment:
We also had issues with the albums not being centered on the overall site. 
We solved it like this:
```
align-items: center; 
align-self: center; 
justify-self: center;
```

Reply:
Thanks!